packages/runtime-dom/src/components/TransitionGroup.ts

```diff
@@ -22,6 +21,7 @@ import {
 } from '@vue/runtime-core'
 import { toRaw } from '@vue/reactivity'
 import { extend } from '@vue/shared'
+import { getTransitionRawChildren } from '../../../runtime-core/src/components/BaseTransition'
```

Comment:
This should be exported from `@vue/runtime-core` - we can't do cross package relative imports.